examples/github-workflow/github-workflow.schema.ncl

```diff
@@ -0,0 +1,2001 @@
+let predicates = import "./lib/predicates.ncl" in
```

Comment:
I wonder if we should insert a heading comment

```nickel
# THIS FILE WAS AUTOMATICALLY GENERATED BY JSON-SCHEMA-TO-NICKEL
# DO NOT EDIT
```

The first motivation wouldn't to prevent editing, but not to scare people off about writing 2kLoC intricate schemas by hand :sweat_smile:

Reply:
That's a very valid concern 😅